config/policy.go

```diff
@@ -807,12 +815,12 @@ func (p *Policy) RouteID() (uint64, error) {
 		hash.WriteInt32(int32(p.Response.Status))
 		hash.WriteStringWithLen(p.Response.Body)
 	default:
-		return 0, errEitherToOrRedirectOrResponseRequired
+		return "", errEitherToOrRedirectOrResponseRequired
 	}
-	return hash.Sum64(), nil
+	return fmt.Sprintf("%x", hash.Sum64()), nil
```

Comment:
```suggestion
	return hex.EncodeToString(hash.Sum64()), nil
```

Reply:
`EncodeToString` expects a []byte

Reply:
oh sorry, perhaps 
```go
hex.EncodeToString(binary.LittleEndian.AppendUint64(nil, hash.Sum64()))
```

Reply:
actually we can just use `hash.Sum` since it returns bytes.